dvc/stage/__init__.py

```diff
@@ -32,6 +32,35 @@
 INVALID_STAGENAME_CHARS = set(string.punctuation) - {"_", "-"}
 
 
+class ExternalOutputsError(DvcException):
+    pass
+
+
+def _check_no_externals(outs):
+    from urllib.parse import urlparse
+
+    # NOTE: preventing users from accidentally using external outputs. See
+    # https://github.com/iterative/dvc/issues/1545 for more details.
+
+    def _is_external(out):
+        # NOTE: if user uses `remote://` notation, he clearly knows what he is
+        # doing and we shouldn't error-out.
```

Comment:
```suggestion
        # NOTE: on `remote://` notation, we shouldn't error-out.
```

Reply:
@skshetry The new comment doesn't provide any hint on `why` though :slightly_smiling_face:

Reply:
@efiop, then let's make it gender-neutral. :smile: